package.json

```diff
@@ -27,9 +27,12 @@
     "precompile": "gts clean"
   },
   "dependencies": {
-    "google-gax": "^3.5.2"
+    "google-gax": "3.5.2",
+    "protobufjs": "7.1.2"
```

Comment:
we could make `google-gax` export `protobufjs` so that there's not a risk that you are using a different protobufjs than each other.

Reply:
`google-gax` already exports `protobufjs`, I added on the main `package.json` file by mistake. I think the right place to added this is on `samples/package.json` file, as for final users seems more intuitive to import `protobufjs` directly from the original package than exposing from `google-gax'.

Also I recently changed to fixed version due to a CI bug that I was facing where TS compilation was failing after release `7.2.3` this week. [See this run](https://github.com/googleapis/nodejs-bigquery-storage/actions/runs/4545486423/jobs/8012881744?pr=328)